### PR TITLE
Moved AdditionalCapabilities out from VM profile to VMSS and updated SKUs API to support zonal capabilities

### PR DESCRIPTION
**Pull request by robhat1**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5937 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5937/commits/d8c4a9af8e66ecbc5ee99b37d9be636279c40bc0

**Comment by dsgouda** [comment]
startbuild

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -7649,10 +7649,6 @@
           "$ref": "#/definitions/VirtualMachineScaleSetStorageProfile",
           "description": "Specifies the storage settings for the virtual machine disks."
         },
-        "additionalCapabilities": {
```
Comment (dsgouda) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: This is a breaking change for a stable version, why is this needed

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 1 above)
Comment (robhat1) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: This property was there for a feature in private preview and based on discussion between the dev team/PM teams from compute and storage + folks from SDK, there was an agreement that we want to change this to provide better user experience from API stand-point.

Comment (dsgouda) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: I understand, but this is a change to a stable version which has already been published, something that is not allowed. Was this approved by ARM

Comment (robhat1) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Have we already published powershell/CLIs and other tools based on this API-version? My understanding was that it is still not done and our team is still pushing changes for this new api-version.

Comment (dsgouda) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Confirmed offline that this version has not been released in SDKs, this is good to go

**Review by dsgouda — APPROVED** [review]
LGTM
